components/modal/ModalLogin.vue

```diff
@@ -1,24 +1,24 @@
 <template>
 	<div>
 		<div class="c-modalRegister" v-if="modalTypeProps === 'modalRegister'">
-			<form @submit.prevent="submitForm">
```

Comment:
i would leave @submit.prevent="submitForm" since its preventing submitting form twice i think also
leave submitForm function see bottom for more instructions